### PR TITLE
Blogger Experience - Initial component implementation of onboarding video UI

### PR DESCRIPTION
**Pull request by blackjackkent**
#### Changes proposed in this Pull Request

* This PR adds a component file that will contain the implementation of the onboarding video UI described in pdgK6S-ck-p2. It does not currently load the UI anywhere, but will be usable in the work going on in https://github.com/Automattic/wp-calypso/issues/57326
image

#### Testing instructions

* N/A

Related to #56888

### CODE REVIEW

File: client/my-sites/customer-home/cards/education/onboarding-video-ui/index.jsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+					<div className="onboarding-video-ui__chapters">
+						<div className="onboarding-video-ui__chapter">
+							1. { translate( 'Set up your blog in 5 steps' ) }{ ' ' }
+							<span className="onboarding-video-ui__duration">01:45</span>{ ' ' }
+							<span className="onboarding-video-ui__completed">
+								<Gridicon icon="checkmark" size={ 12 } />
+							</span>
+						</div>
+						<div className="onboarding-video-ui__chapter">
+							2. { translate( 'Write a blog post' ) }{ ' ' }
```
Comment (a8ci18n) [inline client/my-sites/customer-home/cards/education/onboarding-video-ui/index.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **27** times:
`translate( 'Write a Blog Post' )` *ES Score: 6*

File: client/my-sites/customer-home/cards/education/onboarding-video-ui/index.jsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+						</div>
+						<div className="onboarding-video-ui__chapter">
+							3. { translate( 'How to edit in Wordpress' ) }{ ' ' }
+							<span className="onboarding-video-ui__duration">01:14</span>{ ' ' }
+							<span className="onboarding-video-ui__completed">
+								<Gridicon icon="checkmark" size={ 12 } />
+							</span>
+						</div>
+						<div className="onboarding-video-ui__chapter">
+							4. { translate( 'Social icons' ) }{ ' ' }
```
Comment (a8ci18n) [inline client/my-sites/customer-home/cards/education/onboarding-video-ui/index.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **107** times:
`translate( 'Social Icons' )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/blogger-experience-onboarding-video-ui)

File: client/my-sites/customer-home/cards/education/onboarding-video-ui/index.jsx [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+							</span>
+							<div className="onboarding-video-ui__active-video-content">
+								<p>
+									{ translate(
+										"You're about to embark on the journey of starting a new blog. We'll run through five steps to make sure that you love the way your blog looks so you can feel proud to share it with others."
+									) }{ ' ' }
+								</p>
+								<button type="button" className="onboarding-video-ui__play-button">
+									<Gridicon icon="play" size={ 12 } />
+									{ translate( 'Play video' ) }
```
Comment (a8ci18n) [inline client/my-sites/customer-home/cards/education/onboarding-video-ui/index.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **50** times:
`translate( 'Play Video' )` *ES Score: 11*

**Comment by kwight** [comment]
A few quick first notes:
* This needs to be a standalone component, that has nothing to do with My Home (meaning, it could be pulled in and used anywhere). I'd expect it to be in `/client/components`.
* Only the card from #57326 should be within My Home code.
* We can remove the feature flag – since we're launching quickly, we'll need to rely on #57326 and its calypso.live link for testing.

**Comment by blackjackkent** [comment]
As discussed above - updated the component to be standalone in `client/components` and removed feature flags and initial references.

File: client/components/onboarding-video-ui/index.jsx [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+					<div className="onboarding-video-ui__chapters">
+						<div className="onboarding-video-ui__chapter">
+							1. { translate( 'Set up your blog in 5 steps' ) }{ ' ' }
+							<span className="onboarding-video-ui__duration">01:45</span>{ ' ' }
+							<span className="onboarding-video-ui__completed">
+								<Gridicon icon="checkmark" size={ 12 } />
+							</span>
+						</div>
+						<div className="onboarding-video-ui__chapter">
+							2. { translate( 'Write a blog post' ) }{ ' ' }
```
Comment (a8ci18n) [inline client/components/onboarding-video-ui/index.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **27** times:
`translate( 'Write a Blog Post' )` *ES Score: 6*

File: client/components/onboarding-video-ui/index.jsx [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+						</div>
+						<div className="onboarding-video-ui__chapter">
+							3. { translate( 'How to edit in Wordpress' ) }{ ' ' }
+							<span className="onboarding-video-ui__duration">01:14</span>{ ' ' }
+							<span className="onboarding-video-ui__completed">
+								<Gridicon icon="checkmark" size={ 12 } />
+							</span>
+						</div>
+						<div className="onboarding-video-ui__chapter">
+							4. { translate( 'Social icons' ) }{ ' ' }
```
Comment (a8ci18n) [inline client/components/onboarding-video-ui/index.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **107** times:
`translate( 'Social Icons' )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/blogger-experience-onboarding-video-ui)

File: client/components/onboarding-video-ui/index.jsx [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+							</span>
+							<div className="onboarding-video-ui__active-video-content">
+								<p>
+									{ translate(
+										"You're about to embark on the journey of starting a new blog. We'll run through five steps to make sure that you love the way your blog looks so you can feel proud to share it with others."
+									) }{ ' ' }
+								</p>
+								<button type="button" className="onboarding-video-ui__play-button">
+									<Gridicon icon="play" size={ 12 } />
+									{ translate( 'Play video' ) }
```
Comment (a8ci18n) [inline client/components/onboarding-video-ui/index.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **50** times:
`translate( 'Play Video' )` *ES Score: 11*

**Comment by kwight** [comment]
@blackjackkent Let's just call it `videos-ui`, not `onboarding-video-ui` – the component has nothing to do with onboarding.

**Comment by blackjackkent** [comment]
> @blackjackkent Let's just call it `videos-ui`, not `onboarding-video-ui` – the component has nothing to do with onboarding.

Good call - updated!

File: client/components/videos-ui/index.jsx [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+					<div className="videos-ui__chapters">
+						<div className="videos-ui__chapter">
+							1. { translate( 'Set up your blog in 5 steps' ) }{ ' ' }
+							<span className="videos-ui__duration">01:45</span>{ ' ' }
+							<span className="videos-ui__completed">
+								<Gridicon icon="checkmark" size={ 12 } />
+							</span>
+						</div>
+						<div className="videos-ui__chapter">
+							2. { translate( 'Write a blog post' ) }{ ' ' }
```
Comment (a8ci18n) [inline client/components/videos-ui/index.jsx:68]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **27** times:
`translate( 'Write a Blog Post' )` *ES Score: 6*

File: client/components/videos-ui/index.jsx [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+						</div>
+						<div className="videos-ui__chapter">
+							3. { translate( 'How to edit in Wordpress' ) }{ ' ' }
+							<span className="videos-ui__duration">01:14</span>{ ' ' }
+							<span className="videos-ui__completed">
+								<Gridicon icon="checkmark" size={ 12 } />
+							</span>
+						</div>
+						<div className="videos-ui__chapter">
+							4. { translate( 'Social icons' ) } <span className="videos-ui__duration">03:18</span>{ ' ' }
```
Comment (a8ci18n) [inline client/components/videos-ui/index.jsx:82]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **107** times:
`translate( 'Social Icons' )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/blogger-experience-onboarding-video-ui)

File: client/components/videos-ui/index.jsx [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+							</span>
+							<div className="videos-ui__active-video-content">
+								<p>
+									{ translate(
+										"You're about to embark on the journey of starting a new blog. We'll run through five steps to make sure that you love the way your blog looks so you can feel proud to share it with others."
+									) }{ ' ' }
+								</p>
+								<button type="button" className="videos-ui__play-button">
+									<Gridicon icon="play" size={ 12 } />
+									{ translate( 'Play video' ) }
```
Comment (a8ci18n) [inline client/components/videos-ui/index.jsx:101]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **50** times:
`translate( 'Play Video' )` *ES Score: 11*

File: client/components/videos-ui/style.scss [hunk 10]
```diff
@@ -0,0 +1,71 @@
+.onboarding-video-ui {
```
Comment (kwight) [inline client/components/videos-ui/style.scss]: These would all need to be changed too.

File: client/components/videos-ui/style.scss [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+		background: #151b1e;
+		border-bottom: 1px solid rgba( 255, 255, 255, 0.1 );
+
+		.onboarding-video-ui__header-links {
+			padding: 24px;
+			display: flex;
+			flex-direction: row;
+			align-items: center;
+			justify-content: space-between;
+			font-size: 0.875rem;
```
Comment (kwight) [inline client/components/videos-ui/style.scss:14]: There should be variables for these (I don't think we should ever be hard-coding literal font size values). Just use the one closest. 👍

File: client/components/videos-ui/style.scss [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+			display: flex;
+			justify-content: space-around;
+			padding: 80px 30px;
+			li {
+				list-style: none;
+				margin-bottom: 14px;
+				font-size: 0.875rem;
+			}
+			h2 {
+				font-family: 'Recoleta', 'Noto Serif', Georgia, 'Times New Roman', Times, serif;
```
Comment (kwight) [inline client/components/videos-ui/style.scss:39]: Are these values needed? Is there another way to use existing values? (I honestly don't know, but I'm assuming there are app-wide declarations, and a way to trigger them.)

**Comment by kwight** [comment]
@blackjackkent Just add a screenshot to the description, then go a head and merge – you can make followup issues from my last notes, and we'll go from there. 👍

**Review by kwight — APPROVED** [review]
This is fine for us to get something in place, then start tweaking. 👍

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/6915723

Thank you @blackjackkent for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/6915723).

**Pull request merged**